content/sensu-go/6.1/plugins/supported-integrations/wavefront.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Wavefront integration"
+linkTitle: "Wavefront"
+description: "Use the Sensu Wavefront Handler plugin to integrate Sensu with your existing Wavefront workflows. Read about the features of Sensu's Wavefront integration and learn how to get the plugin."
+version: "6.1"
+product: "Sensu Go"
+menu: 
+  sensu-go-6.1:
+    parent: supported-integrations
+---
+
+The Sensu Wavefront Handler plugin is a Sensu [handler][1] that sends Sensu metrics to Wavefront via a proxy, which allows you to store, instrument, and visualize Sensu metrics data in an Wavefront database.
+
+{{% notice note %}}
+**NOTE**: The Sensu Wavefront Handler plugin is an example of Sensu's status and metrics processing and storage integrations.
+To find more integrations, search [Bonsai, the Sensu asset index](https://bonsai.sensu.io/).
+{{% /notice %}}
+
+## Features
+
+- Transform metrics to Wavefront format: extract and transform the metrics you collect from different sources like Graphite, OpenTSDB, Nagios, and Influx and populate them into Wavefront.
+- Specify additional tags to include when processing metrics.
```

Comment:
This feature of the handler made me think of `output_metric_tags` functionality we recently added to Sensu itself. 

If it is the case that `output_metrics_tags` "just works" with any of our existing handlers, we may want to include a bulletpoint like this one with a link to the relevant documentation on each TSDB integration page.

Reply:
I asked about this in #engineering